README.md

```diff
@@ -43,13 +43,13 @@ pak::pkg_install(pkg_list)
 
 ### Use `renv` to recreate our project library
 
-Another option is to use the [`renv` package](https://rstudio.github.io/renv/index.html) to replicate our exact project library. `renv` will create a private, project-specific library that is separate from your personal library of packages. This would be a good option if, for example, you have another project that relies on a specific version of a package and you don't want to mess with it by upgrading, downgrading, etc.
+Another option is to use the [`renv` package](https://rstudio.github.io/renv/index.html) to replicate our exact project library. `renv` is a package that uses a [snapshot and restore](https://environments.rstudio.com/snapshot.html) strategy to create **r**eproducible **env**nvironments for R projects. `renv` will create a private, project-specific library that is separate from your personal library of packages. This would be a good option if, for example, you have another project that relies on a specific version of a package and you don't want to mess with it by upgrading, downgrading, etc.
```

Comment:
FWIW, standard tidyverse / r-lib "house style" does not format package names as code. So we do not put backticks around "renv" here.

https://style.tidyverse.org/documentation.html#r-code

> Do not use code font for package names. If the package name is ambiguous in the context, disambiguate with words, e.g. “the foo package”. Do not capitalize the function name if it occurs at the start of a sentence.

I can imagine there are many instances of this already, but let's just start fixing it as we touch things.

Reply:
Ah okay good to know, I will remove the backticks. I was referencing an RStudio Style Guide google doc (not sure if it's okay to include the link here?) which showed a package in backticks:

> If referring to the actual package name in R (e.g., code that the user may type), then use the actual capitalization in the package name (which is usually lowercase, but not always), even, in most cases,  if the package name comes at the start of the sentence. Example: `ggvis` or `ggplot2`.

I made sure to be consistent with surrounding packages with backticks throughout the bookdown so they should all have them -- should I change them all in a separate PR so it's one big switch?

Reply:
Let's just fix when we are touching something anyway. At least for now. Maybe later one last big sweep would make sense.